modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -73,8 +73,8 @@ sub fetch_input {
 
 	# check there are no seq_region changes in the existing species
 	my $list_cmd = "perl $list_genomes_script $metadata_script_options";
-	my $list_run = $self->run_command($list_cmd);
-	my @release_genomes = split( /\s+/, $list_run->out );
+	my $list_out = $self->get_command_output($list_cmd);
+	my @release_genomes = split( /\s+/, $list_out );
```

Comment:
Doesn't `get_command_output` detect `wantarray` and do this splitting automatically?
```suggestion
	my @release_genomes = $self->get_command_output($list_cmd);
```

Reply:
`get_command_output` splits on new-lines (`\n`). Would that work here (the pattern currently is `\s+`) ?

Reply:
Yes, I've just checked and splitting on newline is fine for this case.